### ISSUE TITLE
Request for Checkpoints on Cifar100 Dataset.

### ISSUE THREAD

**Issue by Yushu-Li** [LINs-lab/ttab#12]
Hi, thanks for your excellent work!
To ensure the accuracy of my replication efforts, I am kindly requesting access to the pretrained checkpoints you used in your experiments on the Cifar100 dataset. I have tried to use "python ssl_pretrain.py --data-name cifar100 --model-name resnet26" to pre-train a Cifar100 model, but it can not reproduce the results in the paper.

**Comment by MarcellusZhao** [comment]
Hello,

Thanks for your interest in our work!

Please check this gdrive [link](https://drive.google.com/drive/u/0/folders/1KBbcNB6KR5Fqi6iueASLb85LxpkraX3K), I have uploaded the checkpoint for CIFAR100.

**Comment by Yushu-Li** [comment]
Hi Marcellus,

Thanks for your quick reply. I will use this pre-trained model to reproduce the results. 

Best
